applications/serial_lte_modem/README.rst

```diff
@@ -197,6 +198,158 @@ The different command options for <cmd> are listed below:
 * AT#XFTP="get",<file>
 * AT#XFTP="put",<file>[<datatype>,<data>]
 
+HTTP Client AT Commands
+***********************
+
+The following proprietary HTTP Client AT commands and unsolicited notifications are used in this application:
+
+.. list-table::
+   :align: left
+   :header-rows: 1
+
+   * - HTTP Client AT Commands
+     - Description
+   * - AT#XHTTPCCON=<op>[,<host>,<port>[,<sec_tag>]]
+     - Connect to/disconnect from a HTTP server.
+   * - AT#XHTTPCREQ=<method>,<resource>,<header>[,<payload_length>]
+     - Send HTTP request to the HTTP server.
+   * - #XHTTPCRSP=<byte_received>,<state><CR><LF><response>
+     - Indicate that a part of HTTP response was received.
+
+AT#XHTTPCCON
+============
+
+The AT#XHTTPCCON command connect to/disconnect from a HTTP server.
+
+.. code-block:: none
+
+   AT#XHTTPCCON=<op>[,<host>,<port>[,<sec_tag>]]
+
+.. csv-table::
+    :header: Command Parameters, Data Type, Description
+
+    op, integer, "| 1: Connect to HTTP host
+    | 0: Disconnect from HTTP host"
+    host, string, Domain name of HTTP server.
+    port, integer, TCP port number on which HTTP server is listening.
+    sec_tag, integer, Security tag to be used for secure session.
+
+The #XHTTPCCON unsolicited notification indicates the state of connection.
+
+.. code-block:: none
+
+   #XHTTPCCON=<state>
+
+.. csv-table::
+    :header: Notification Parameters, Data Type, Description
+
+    state, integer, "| 1: Connected to HTTP host
+    | 0: Disconnected from HTTP host"
+
+The example below connects to a HTTP server:
+
+.. code-block:: none
+
+   AT#XHTTPCCON=1,"postman-echo.com",80
+   #XHTTPCCON:1
+   OK
+
+AT#XHTTPCREQ
+============
+
+The AT#XHTTPCREQ sends HTTP request to the HTTP server.
+
+.. code-block:: none
+
+   AT#XHTTPCREQ=<method>,<resource>,<header>[,<payload_length>]
+
+.. csv-table::
+    :header: Command Parameters, Data Type, Description
+
+    method, string, "Request method string."
+    resource, string, "Target resource to apply the request"
```

Comment:
```suggestion
    resource, string, "Target resource to apply the request."
```

Reply:
Is "apply" the correct term here? Maybe "Target resource to handle the request."?

Reply:
Yes, handle should be better. I've replaced "apply" by "handle".